serviceobjects/src/main/java/TripDataSO.java

```diff
@@ -0,0 +1,99 @@
+import java.sql.Date;
+import java.util.Collection;
+
+public class TripDataSO {
+    private int id;
+    private RouteSO route;
+    private int seatsLeft;
+    private byte isCancelled;
+    private byte isLate;
+    private Date date;
+    private Collection<TicketSO> ticketsFrom;
```

Comment:
please init this collection or modify get-method.
Collection property shouldn't be null